isaric/parsers/predicovid.toml

```diff
@@ -392,7 +425,7 @@
   is_present = { field = "QM9", values = { 1 = true, 0 = false } }
 
 [[observation]]
-  name = "muscle_aches"
+  name = "musculoskeletal_pain"
```

Comment:
interesting .. that there are differences between 'muscle aches' and 'musculoskeletal pain'